docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+author: chgill-MSFT
+ms.author: chgill
+ms.date: 09/17/20
+ms.topic: conceptual
+---
+
+# Package authoring best practices
+
+This guidance is intended to give NuGet.org package authors a light-weight end to end reference for creating and publishing high quality packages. It will primarily focus on package-specific best practices such as metadata and packing. For more in depth suggestions for building high quality libraries, see the .NET [Open-source library guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/).
+
+## Types of recommendations
+
+Each article presents four types of recommendations: **Do**, **Consider**, **Avoid**, and **Do not**. The type of recommendation indicates how closely it should be followed.
+
+You should almost always follow a **Do** recommendation. For example:
+
+✔️ DO include a short description for your package that describes what it's for.
+
+On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule and you shouldn't feel bad about not following the guidance:
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/reference/id-prefix-reservation).
+
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
+
+❌ AVOID NuGet package references that demand an exact version.
+
+And finally, **Do not** recommendations indicate something you should almost never do:
+
+❌ DO NOT use the `LicenseUrl` metadata property.
+
+## Framework targeting
+
+Authors should consider making their packages as inclusive as possible if they would like to maximize the number of potential consumers and overall impact in the .NET ecosystem.
+
+Highly inclusive packages are cross-platform and compatible with as many frameworks as possible. Today, that means using .NET Standard. .NET Standard is a specification of .NET APIs that are available on all .NET implementations.
+
+✔️ DO start with including a `netstandard2.0` target.
+> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.
+
+See the .NET [Cross-platform targeting guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting) for more advanced suggestions regarding cross-platform and multitargeting scenarios.
+
+## Package metadata
+
+Metadata is a foundational component of any NuGet package. The quality of your metadata can vastly influence the discoverability and usability of your package.
+
+In Visual Studio, the recommended way to specify package metadata is to go Project > [Project Name] Properties > Package.
+
+Package metadata elements can also be specified directly in the project file (.csproj).
```

Comment:
Should we be language agnostic and include F#'s .fsproj project files?

Reply:
TIL F# has an .fsproj file instead of a .csproj file 😂

Reply:
Consider linking to documentation where possible. For example here: https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package-msbuild#set-properties